### PR TITLE
Added doc: host setup - disable lvmetad

### PR DESCRIPTION
**Pull request by alexzhc**
This doc update provides instruction to fix the issue where in centos 7 and ubuntu bionic, one must run `pvscan --cache` before `vgdispaly, lvdisplay, lvs` to see Piraeus volume LVs

### CODE REVIEW

File: doc/host-setup.md
Comment (WanzenBug) [inline doc/host-setup.md:150]:
```suggestion
## Disable Lvmetad on CentOS 7 and Ubuntu 18.04
```

File: doc/host-setup.md
Comment (WanzenBug) [inline doc/host-setup.md:152]: This link requires logging in with a red hat account. I think there is enough information so we can remove it.

File: doc/host-setup.md
Comment (WanzenBug) [inline doc/host-setup.md:160]:
```suggestion
Follow below steps to disable lvmetad completely:
```

There is a trailing whitespace here

File: doc/host-setup.md
Comment (WanzenBug) [inline doc/host-setup.md:173]:
```suggestion
If the root partition already uses LVM, you also need to update the initial ram-disk:
```

**Review by WanzenBug — CHANGES_REQUESTED** [review]
Instructions look good. Please fix the CI issues (trailing whitespaces) and see about `--signoff` on the commit so the DCO bot is happy.

File: doc/host-setup.md
Comment (kvaps) [inline doc/host-setup.md:173]: Is it really necessary step? You already disabled `lvm2-lvmetad.service` and `lvm2-lvmetad.socket`, and configured `lvm.conf` to not use it.

Why do we need any changes on initramfs phase?

File: doc/host-setup.md
Comment (kvaps) [inline doc/host-setup.md:166]: Why to mask without disabling lvm2-lvmetad.socket?

File: doc/host-setup.md
Comment (kvaps) [inline doc/host-setup.md:156]: Haven't we considering passing through `/etc/lvm.conf` and `/run/lvm` from host into container to avoid such problems?

**Review by kvaps — COMMENTED** [review]
Just few suggestions

File: doc/host-setup.md
Comment (alexzhc) [reply doc/host-setup.md]: It is recommended by Redhat to load lvm.conf properly during boot, specifically for root on lvm. Apart from that, in CentOS 7 without updated intramfs, commends such as vgdisplay will still complain about missing lvmetad.

File: doc/host-setup.md
Comment (alexzhc) [reply doc/host-setup.md:164]: It is also recommend by RedHat's article "[https://access.redhat.com/solutions/2053483] https://access.redhat.com/solutions/2053483"

File: doc/host-setup.md
Comment (alexzhc) [reply doc/host-setup.md]: Yes, I have another incoming PR to support lvmetad just for centos 7 and ubuntu bionic as we also hear from some conservative users who do not want to turn lvmetad off. But so far, disabling lvmetad is the simplest solution and is consistent with the lvm trend in centos 8 and ubuntu focal.

Comment (alexzhc) [reply doc/host-setup.md]: Passing `/etc/lvm.conf` and `/run/lvm` is not adequate to enable lvm binary communicate to os-level lvmetad daemon from within `piraeus-ns-node` container.  Only lvm binary lvm2-2.02 binary in centos/ubi 7, ubuntu bionic, and debian strech base images still have support for lvmetad ( they still having `use_lvmetad = 1` in lvm.conf).  `piraeus-sever` is currently based on debian bullseye (presviously buster) image which uses lvm2-2.03 that has deprecated lvmetad support. Therefore, to support lvmetad, `piraeus-server` needs to have a version based on debian stretch (or ubuntu bionic or centos 7) to be able to install lvm2-2.02.  Apart from that, /run/udev is also needed for lvmetad.

Comment (alexzhc) [reply doc/host-setup.md]: Unfornately, linbit does not maintain a repo for debian stretch any more. Therefore, I managed to rebase `piraeus-server`to ubuntu bionic image, and successfully had lvm cmd updating lvmetad in os.

**Review by WanzenBug — CHANGES_REQUESTED** [review]
Please:
* Install https://pre-commit.com/ and run `pre-commit run --source=811e973 --to-ref=HEAD` and commit any changes, that should fix the CI check.
* Please run `git commit --amend --signoff` and force-push the new commit to fix the DCO bot.

I can't merge this as long as one the checks doesn't pass

**Comment by alexzhc** [comment]
> Please:
> 
>     * Install https://pre-commit.com/ and run `pre-commit run --source=811e973 --to-ref=HEAD` and commit any changes, that should fix the CI check.
> 
>     * Please run `git commit --amend --signoff` and force-push the new commit to fix the DCO bot.
> 
> 
> I can't merge this as long as one the checks doesn't pass

I've done as you instructed, please see if it can be merged.

**Comment by WanzenBug** [comment]
> I've done as you instructed, please see if it can be merged.

Still doesn't let me merge:
* https://github.com/piraeusdatastore/piraeus-operator/runs/6083699465?check_suite_focus=true
* https://github.com/piraeusdatastore/piraeus-operator/pull/296/checks?check_run_id=6083684183

**Comment by alexzhc** [comment]
I can fix the signed off. But for https://github.com/piraeusdatastore/piraeus-operator/runs/6083699465?check_suite_focus=true
```
Fix End of Files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook
```

I've already double-checked newlines in `host-setup.md`, and found no trailing whitespaces. Is there any way to have "pre-commit" just fix the End of Files as it sees instead of warn each time?

**Comment by alexzhc** [comment]
@WanzenBug I've fixed signoffs. Running pre-commit on `--all-files` passed, please try merge again. 
```
$ pre-commit run -v --source=811e973 --to-ref=HEAD --all-files
trim trailing whitespace.................................................Passed
- hook id: trailing-whitespace
- duration: 0.04s
fix end of files.........................................................Passed
- hook id: end-of-file-fixer
- duration: 0.04s
check yaml...........................................(no files to check)Skipped
- hook id: check-yaml
check for added large files..............................................Passed
- hook id: check-added-large-files
- duration: 0.06s
gofumpt..............................................(no files to check)Skipped
- hook id: gofumpt
golangci-lint........................................(no files to check)Skipped
- hook id: golangci-lint
go-unit-tests........................................(no files to check)Skipped
- hook id: go-unit-tests
generated deep-copy code must be up-to-date..........(no files to check)Skipped
- hook id: generate-deep-copy
generated crds must be up-to-date....................(no files to check)Skipped
- hook id: generate-crds
generate helm values for CN users....................(no files to check)Skipped
- hook id: generate-cn-values
generate full yaml deployment from helm templates....(no files to check)Skipped
```

File: doc/host-setup.md
Comment (WanzenBug) [inline doc/host-setup.md:182]: Weird that this wasn't caught by your `pre-commit` run, but this still fails here. Just remove this line, I think thats the last blocker

**Review by WanzenBug — CHANGES_REQUESTED** [review]
Sorry, but this still fails the pre-commit check. Just remove the offending extra blank line, and it should pass.

Afterwards, please run `git rebase -i 811e973de498120d525d9e8e7ed7ad9e83ade82c` and replace `fixup` all the commits introduced by these reviews, looking like this:
```
pick ea8aa96 Added doc: host setup - disable lvmetad
fixup 6ded6da Update doc/host-setup.md
fixup 8c1857e Update doc/host-setup.md
fixup dadef0e Update doc/host-setup.md
...
```

This will collapse all commits into a single one, so the git history looks clean.

File: doc/host-setup.md
Comment (alexzhc) [reply doc/host-setup.md]: As per vscode markdownlint suggestion, I added blank lines before and after code block. `git commit` passed with no error per `pre-commit`: 

```
 git commit -s -m 'clear syntax errors posted by markdownlint'
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check yaml...........................................(no files to check)Skipped
check for added large files..............................................Passed
gofumpt..............................................(no files to check)Skipped
golangci-lint........................................(no files to check)Skipped
go-unit-tests........................................(no files to check)Skipped
generated deep-copy code must be up-to-date..........(no files to check)Skipped
generated crds must be up-to-date....................(no files to check)Skipped
generate helm values for CN users....................(no files to check)Skipped
generate full yaml deployment from helm templates....(no files to check)Skipped
[master be83e4e] clear syntax errors posted by markdownlint
 1 file changed, 4 insertions(+), 2 deletions(-)
```
Please see if it can be merged.

**Review by WanzenBug — APPROVED** [review]
LGTM

**Pull request merged**
